src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5282,19 +5281,48 @@ void wm_vuldet_create_and_send_message(vu_task_ctx_t* task, task_status TASK_STA
     }
 }
 
-void wm_vuldet_on_demand_scan(vu_task_ctx_t* current_task, wm_vuldet_t* vuldet) {
+bool wm_vuldet_on_demand_scan(vu_task_ctx_t* current_task, wm_vuldet_t* vuldet) {
```

Comment:
I would unify with wm_vuldet_scan_on_demand.

Reply:
The name were unified